backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandra.java

```diff
@@ -133,7 +133,7 @@ private static String buildSpecificImageDiscriminator() {
 
     @SuppressWarnings("resource")
     public DockerCassandra() {
-        this("cassandra_3_11_10-" + buildSpecificImageDiscriminator(), AdditionalDockerFileStep.IDENTITY);
+        this("cassandra_3_11_15-" + buildSpecificImageDiscriminator(), AdditionalDockerFileStep.IDENTITY);
```

Comment:
Why not switch to Cassandra 4.1.x (in another PR maybe?)

Reply:
https://github.com/linagora/james-project/issues/4822